cfg/supervision.cfg

```diff
@@ -19,7 +19,7 @@ SEGMENTS {
     CODE:     load = ROM,            type = ro,  define   = yes;
     RODATA:   load = ROM,            type = ro,  define   = yes;
     DATA:     load = ROM, run = RAM, type = rw,  define   = yes;
-    FFF0:     load = ROM,            type = ro,  offset   = $7FF0;
+    FFF0:     load = ROM,            type = ro,  offset   = $7FEA;
```

Comment:
Sure that the segment should still be named `FFF0` if its offset is now `$7FEA`? Is it really intended to have the same offset twice (see segment `VECTOR`)?

Reply:
You are perfectly right on the name.